src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -0,0 +1,15 @@
+package racingcar.domain
+
+import racingcar.constant.ERROR_CARS_CREATE
+
+class Cars(names: List<RacingCar>) {
+    private var _value: List<RacingCar> = names
+    val value: List<RacingCar> get() = _value
```

Comment:
```suggestion
class Cars(val names: List<RacingCar>) {
```
backing property는 클래스 내부에서만 값을 변경하고, 외부에서는 값에 접근만 하기 위해 사용하는 기법입니다.
Cars 클래스에는 _value를 사용하는 곳이 없습니다.
또한, List와 같은 컬렉션은 자체적으로 add / remove 등 데이터 제어 함수들을 지원하기 때문에, var을 사용하지 않는 것을 권장합니다.

Reply:
아 이런부분은 생각하지 못했네요 감사합니다